<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportVisual</class>
 <widget class="QMainWindow" name="ReportVisual">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>551</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>VisualReport v.0.1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recoverdrake.qrc">
    <normaloff>:/Imagenes/modelo.png</normaloff>:/Imagenes/modelo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>15</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Principal</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionVista_Previa"/>
    <addaction name="separator"/>
    <addaction name="actionAbrir_desde_BD"/>
    <addaction name="actionGuardar_en_BD"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuInsertar">
    <property name="title">
     <string>Insertar</string>
    </property>
    <addaction name="actionCrear_seccion"/>
    <addaction name="separator"/>
    <addaction name="actionRectangulo_redondeado"/>
    <addaction name="actionTexto"/>
    <addaction name="actionImagen"/>
    <addaction name="actionLinea"/>
    <addaction name="actionCampo"/>
    <addaction name="actionCampo_Relacional"/>
    <addaction name="actionCodeBar"/>
    <addaction name="actionParametro"/>
    <addaction name="actionAcumulador"/>
   </widget>
   <widget class="QMenu" name="menuHerramientas">
    <property name="title">
     <string>Herramientas</string>
    </property>
    <addaction name="actionBlock"/>
    <addaction name="actionDestruir_bloque"/>
    <addaction name="separator"/>
    <addaction name="actionConfigurar_modelo"/>
    <addaction name="separator"/>
    <addaction name="actionVer_contorno_objetos"/>
    <addaction name="actionLimitar_vista_previa"/>
    <addaction name="separator"/>
    <addaction name="actionBorrar"/>
    <addaction name="actionMostrar_ocultar_lista_de_campos"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuInsertar"/>
   <addaction name="menuHerramientas"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>13</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionCrear_seccion"/>
   <addaction name="actionRectangulo_redondeado"/>
   <addaction name="actionTexto"/>
   <addaction name="actionImagen"/>
   <addaction name="actionLinea"/>
   <addaction name="actionCampo"/>
   <addaction name="actionCampo_Relacional"/>
   <addaction name="actionCodeBar"/>
   <addaction name="actionParametro"/>
   <addaction name="actionAcumulador"/>
   <addaction name="separator"/>
   <addaction name="actionDestruir_bloque"/>
   <addaction name="actionBlock"/>
   <addaction name="actionBorrar"/>
   <addaction name="separator"/>
   <addaction name="actionSalir"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionNuevo"/>
   <addaction name="separator"/>
   <addaction name="actionAbrir"/>
   <addaction name="actionGuardar"/>
   <addaction name="separator"/>
   <addaction name="actionAbrir_desde_BD"/>
   <addaction name="actionGuardar_en_BD"/>
   <addaction name="separator"/>
   <addaction name="actionVista_Previa"/>
   <addaction name="actionConfigurar_modelo"/>
   <addaction name="actionMostrar_ocultar_lista_de_campos"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAlinear_Izquierda"/>
   <addaction name="actionAlinear_Derecha"/>
   <addaction name="actionAlinear_Centro"/>
   <addaction name="actionAlinear_Arriba"/>
   <addaction name="actionAlinear_centro"/>
   <addaction name="actionAlinear_abajo"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Ordenar A~Z</string>
       </property>
       <property name="icon">
        <iconset resource="recoverdrake.qrc">
         <normaloff>:/Imagenes/flechaOut.png</normaloff>:/Imagenes/flechaOut.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QListWidget" name="itemList">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Desmarcar todo</string>
       </property>
       <property name="icon">
        <iconset resource="recoverdrake.qrc">
         <normaloff>:/Imagenes/menos.png</normaloff>:/Imagenes/menos.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Marcar todo</string>
       </property>
       <property name="icon">
        <iconset resource="recoverdrake.qrc">
         <normaloff>:/Imagenes/mas.png</normaloff>:/Imagenes/mas.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Ordenar Z~A</string>
       </property>
       <property name="icon">
        <iconset resource="recoverdrake.qrc">
         <normaloff>:/Imagenes/flechaUp.png</normaloff>:/Imagenes/flechaUp.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/puerta.png</normaloff>:/Imagenes/puerta.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/carpeta.png</normaloff>:/Imagenes/carpeta.png</iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/guardar.png</normaloff>:/Imagenes/guardar.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionConfigurar_modelo">
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/informe.png</normaloff>:/Imagenes/informe.png</iconset>
   </property>
   <property name="text">
    <string>Configurar</string>
   </property>
  </action>
  <action name="actionRectangulo_redondeado">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/rectredondeado.png</normaloff>:/Imagenes/rectredondeado.png</iconset>
   </property>
   <property name="text">
    <string>Rectangulo</string>
   </property>
   <property name="toolTip">
    <string>Rectangulo</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
  </action>
  <action name="actionTexto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/drakfont-mdk.png</normaloff>:/Imagenes/drakfont-mdk.png</iconset>
   </property>
   <property name="text">
    <string>Texto</string>
   </property>
   <property name="toolTip">
    <string>Texto a insertar</string>
   </property>
   <property name="shortcut">
    <string>Alt+T</string>
   </property>
  </action>
  <action name="actionCrear_seccion">
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/archivoIn.png</normaloff>:/Imagenes/archivoIn.png</iconset>
   </property>
   <property name="text">
    <string>Secciones</string>
   </property>
   <property name="toolTip">
    <string>Secciones</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionVista_Previa">
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/print-preview.png</normaloff>:/Imagenes/print-preview.png</iconset>
   </property>
   <property name="text">
    <string>Vista previa</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionImagen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/pintura.png</normaloff>:/Imagenes/pintura.png</iconset>
   </property>
   <property name="text">
    <string>Imagen</string>
   </property>
   <property name="shortcut">
    <string>Alt+I</string>
   </property>
  </action>
  <action name="actionBorrar">
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/bad.png</normaloff>:/Imagenes/bad.png</iconset>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionLinea">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/line.png</normaloff>:/Imagenes/line.png</iconset>
   </property>
   <property name="text">
    <string>Linea</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionCampo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/field.png</normaloff>:/Imagenes/field.png</iconset>
   </property>
   <property name="text">
    <string>Campo</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionBlock">
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/candadoC.png</normaloff>:/Imagenes/candadoC.png</iconset>
   </property>
   <property name="text">
    <string>Crear bloque</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionDestruir_bloque">
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/muro.png</normaloff>:/Imagenes/muro.png</iconset>
   </property>
   <property name="text">
    <string>Destruir bloque</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="actionVer_contorno_objetos">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ver contorno de objetos</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionLimitar_vista_previa">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Limitar vista previa</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionAlinear_Izquierda">
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/shape_align_left.png</normaloff>:/Imagenes/shape_align_left.png</iconset>
   </property>
   <property name="text">
    <string>Alinear Izquierda</string>
   </property>
   <property name="toolTip">
    <string>Alinear Izquierda</string>
   </property>
  </action>
  <action name="actionAlinear_Derecha">
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/shape_align_right.png</normaloff>:/Imagenes/shape_align_right.png</iconset>
   </property>
   <property name="text">
    <string>Alinear_Derecha</string>
   </property>
   <property name="toolTip">
    <string>Alinear Derecha</string>
   </property>
  </action>
  <action name="actionAlinear_Centro">
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/shape_align_center.png</normaloff>:/Imagenes/shape_align_center.png</iconset>
   </property>
   <property name="text">
    <string>Alinear Centro</string>
   </property>
   <property name="toolTip">
    <string>Alinear Centro</string>
   </property>
  </action>
  <action name="actionAlinear_Arriba">
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/shape_align_top.png</normaloff>:/Imagenes/shape_align_top.png</iconset>
   </property>
   <property name="text">
    <string>Alinear Arriba</string>
   </property>
   <property name="toolTip">
    <string>Alinear Arriba</string>
   </property>
  </action>
  <action name="actionAlinear_centro">
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/shape_align_middle.png</normaloff>:/Imagenes/shape_align_middle.png</iconset>
   </property>
   <property name="text">
    <string>Alinear centro</string>
   </property>
   <property name="toolTip">
    <string>Alinear centro</string>
   </property>
  </action>
  <action name="actionAlinear_abajo">
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/shape_align_bottom.png</normaloff>:/Imagenes/shape_align_bottom.png</iconset>
   </property>
   <property name="text">
    <string>Alinear abajo</string>
   </property>
  </action>
  <action name="actionNuevo">
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/icono.png</normaloff>:/Imagenes/icono.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="toolTip">
    <string>Nuevo</string>
   </property>
  </action>
  <action name="actionMostrar_ocultar_lista_de_campos">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/target.png</normaloff>:/Imagenes/target.png</iconset>
   </property>
   <property name="text">
    <string>Mostrar/ocultar lista de campos</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionCodeBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/barcode.png</normaloff>:/Imagenes/barcode.png</iconset>
   </property>
   <property name="text">
    <string>Codigo de Barras</string>
   </property>
   <property name="toolTip">
    <string>Codigo de Barras</string>
   </property>
   <property name="shortcut">
    <string>Alt+B</string>
   </property>
  </action>
  <action name="actionCampo_Relacional">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/field_rel.png</normaloff>:/Imagenes/field_rel.png</iconset>
   </property>
   <property name="text">
    <string>Campo Relacional</string>
   </property>
   <property name="toolTip">
    <string>Campo Relacional</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionParametro">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/parametro.png</normaloff>:/Imagenes/parametro.png</iconset>
   </property>
   <property name="text">
    <string>Parametro</string>
   </property>
   <property name="toolTip">
    <string>Parametro</string>
   </property>
  </action>
  <action name="actionAcumulador">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/ccm3.png</normaloff>:/Imagenes/ccm3.png</iconset>
   </property>
   <property name="text">
    <string>Acumulador</string>
   </property>
   <property name="toolTip">
    <string>Acumulador</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionGuardar_en_BD">
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/save_db.png</normaloff>:/Imagenes/save_db.png</iconset>
   </property>
   <property name="text">
    <string>Guardar en BD</string>
   </property>
   <property name="toolTip">
    <string>Guardar en BD</string>
   </property>
  </action>
  <action name="actionAbrir_desde_BD">
   <property name="icon">
    <iconset resource="recoverdrake.qrc">
     <normaloff>:/Imagenes/upload_db.png</normaloff>:/Imagenes/upload_db.png</iconset>
   </property>
   <property name="text">
    <string>Abrir desde BD</string>
   </property>
   <property name="toolTip">
    <string>Abrir desde BD</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>itemList</tabstop>
 </tabstops>
 <resources>
  <include location="recoverdrake.qrc"/>
 </resources>
 <connections/>
</ui>
